src/common/ty/ty_printer.ml

```diff
@@ -223,12 +223,19 @@ let type_ ?(size=5000) ?(with_comments=true) t =
       | Get t -> fuse [
           Atom "get"; space;
           to_key key;
+          Atom "(";
+          Atom ")";
+          Atom ":";
           type_ ~depth t;
         ]
 
       | Set t -> fuse [
           Atom "set"; space;
           to_key key;
+          Atom "(";
+          type_ ~depth t;
+          Atom ")";
+          Atom ":";
```

Comment:
let's drop the return type. it adds noise and I think we'll either make it a parse error to specify one or only allow `void`. but either way you shouldn't/can't return a value.